server/service/src/sync/api_v6/core.rs

```diff
@@ -8,6 +8,7 @@ use super::*;
 pub(crate) struct SyncApiV6 {
     pub(crate) url: Url,
     pub(crate) sync_v5_settings: SyncApiSettings,
+    pub(crate) sync_v6_version: u32,
```

Comment:
I _think_ this makes the most sense, having `sync_v6_version` separate from the `sync_v5_settings`. Let me know if you think it should be passed in/around differently though?

Reply:
Yep, sounds good, thanks